volumes/short_term_counting_program/README.md

```diff
@@ -256,4 +256,4 @@ category_name|text|name of the data source
 
 - `traffic.artery_traffic_signals` - A lookup view between artery codes and px numbers that have traffic signals. 
 
-
+- `traffic.artery_objectid_pavement_asset` - A lookup view between artery codes and objectid. Used, for example, to link an arterycode to pavement assest information in vz_analysis.gcc_pavement_asset. This view uses the intermediate table `gis_shared_streets.centreline_pavement_180430` which was last updated three years ago and it will be updated via issue [Update pavement assets #620](https://github.com/CityofToronto/bdit_data-sources/issues/620) 
```

Comment:
Ack there is a tiny typo on the word "assets" ...and it would be awesome if there was a period after the link!

Reply:
fixed!

Reply:
Awesome! I have a 1:1 with radumas this aft and we will discuss next steps!!!